### ISSUE TITLE
Reproduce lower performance on sth-sth-v1

### ISSUE THREAD

**Issue by AliceCrystal** [mit-han-lab/temporal-shift-module#96]
Thanks for your great jobs ! When I try to reproduce the result of sth-sth-v1 dataset, I get some
problem. This is my args.txt content:

`Namespace(arch='resnet50', batch_size=20, clip_gradient=20, consensus_type='avg', dataset='something', dense_sample=False, dropout=0.5, epochs=50, eval_freq=1, evaluate=False, flow_prefix='', gpus=[0, 1], img_feature_dim=256, k=3, loss_type='nll', lr=0.01, lr_steps=[20, 40], lr_type='step', modality='RGB', momentum=0.9, no_partialbn=False, non_local=False, num_segments=8, pretrain='imagenet', print_freq=20, resume='', root_log='log', root_model='checkpoint', root_path='', shift=True, shift_div=8, shift_place='blockres', snapshot_pref='', start_epoch=0, store_name='TSM_something_RGB_resnet50_shift8_blockres_avg_segment8_e50', suffix=None, temporal_pool=False, train_list='/media/a0/18A02F4CA02F3024/gt/data/train_videofolder.txt', tune_from=None, val_list='/media/a0/18A02F4CA02F3024/gt/data/val_videofolder.txt', weight_decay=0.0005, workers=16) `

At last, the accuracy of the trained model is 41.6%, I am really confused where is wrong?

**Comment by tonylins** [comment]
Hi, I notice that you are using a smaller batch size (20). You should also reduce the learning rate from 0.01 to 0.01 * 20 / 64 = 0.003125. You also need to enable `--npb` (see the training command in README). Make sure ImageNet pre-trained weight is enabled.

**Comment by AliceCrystal** [comment]
Understand！Thank you very much，I will try again

**Comment by tonylins** [comment]
Hi, may I know the current status of the problem?

**Comment by tonylins** [comment]
Close due to inactivity. Feel free to reopen.
